doc/develop/test/twister.rst

```diff
@@ -699,6 +699,19 @@ required_snippets: <list of needed snippets>
               - cdc-acm-console
               - user-snippet-example
 
+bsim_copy_exe: <True|False> (default False)
```

Comment:
This is adding bsim specifics into twister and test configurations without any additional context. Can this be generalized? Copying files and making them available to tooling sounds like a very common thing.

Reply:
Yes, this is BabbleSim simulator specific option, because to make it possible to run BabbleSim simulation it is necessary to put in one place (`${BSIM_OUT_PATH}/bin`) all necessary executables. And as far as I know other hardware/simulators do not need this extra step.

You ask about more general approach - can you provide some practical use-case? I have an idea how this could be achieved, but at this moment without any other use-case beyond BabbleSim it is not so clear what is expected from this "general" approach. 

What do you think about something like that:
```yaml
# in testcase.yaml
copy_exe:
  destination_directory: "/destination/path/of/copied/exe/"
  copied_exe_name: "new_name_of_copied_exe"
```